styleguide/containers/BaseContainer.jsx

```diff
@@ -13,9 +13,9 @@ export default class BaseContainer extends React.Component
         <p className="usa-font-lead">
           {this.props.summary}
         </p>
-        <div className="usa-grid">
```

Comment:
Okay.  Sounds good.  I put it in because I thought we'd need it, but if we don't then it's fine to remove.